data-model/src/main/java/org/alfresco/repo/content/ContentStore.java

```diff
@@ -344,4 +345,38 @@ default DirectAccessUrl requestContentDirectUrl(String contentUrl, boolean attac
     {
         return Collections.emptyMap();
     }
+
+    /**
+     * Submit a request to send content to archive (offline) state.
+     * If no connector is present or connector is not supporting sending to archive, then {@code false} will be returned.
+     * Specific connector will decide which storage class/tier will be set for content.
+     * This method is experimental and subject to changes.
+     *
+     * @param contentUrl the URL of the content which is to be archived.
+     * @return true when request successful, false when unsuccessful.
+     */
+    @Experimental
+    default boolean requestSendContentToArchive(String contentUrl)
```

Comment:
Should we leave open the option to send params (similar to restore) - eg. in future, S3 Connector implements ability for client to choose between Glacier FR and Glacier DA.

Reply:
Since this is still `@Experimental` I think we can change this in near future (and not block further Cloud Connectors implementations for now). 
On the other hand, there is some of that choice (surely not that much) left on Cloud Connector side (design says about possibility to globally configure storage class for the archive operation.

Reply:
I'd suggest changing now if there is a good chance we might change again in near future, ie. before we remove Experimental for 7.2. I think it's ok if ignored by cloud connector impl for now.